bitflow/io/sources.py

```diff
@@ -338,6 +399,7 @@ def start(self):
 
     def stop(self):
         self.running.value = 0
+        self.listensource.stop()
```

Comment:
unable to call stop function of multiprocessing.process, shared variable self.running.value is used to stop source process